login.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Cantarell:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/pokestyles.css">
+    <title>Pokédex</title>
+</head>
+<body>
+<header>
+    <img src="img/pokeball.svg">
```

Comment:
Suggestion: could use background image instead.

Reply:
We made it an image because it was easier to rotate and position. I don't think it really matters all that much.